python/ray/serve/schema.py

```diff
@@ -130,10 +135,11 @@ class Config:
         extra = Extra.forbid
 
     encoding: Union[str, EncodingType] = Field(
-        default="TEXT",
+        default_factory=_ray_serve_log_encoding,
```

Comment:
```suggestion
        default_factory=lambda: RAY_SERVE_LOG_ENCODING,
```
A little easier to read IMO :)

Reply:
I think this can actually just be `RAY_SERVE_LOG_ENCODING`, no need for the factory

Reply:
Yea, I tried `default= RAY_SERVE_LOG_ENCODING` initially, but patching and reloading modules correctly in pytest is almost impossible 😅

Reply:
Got it 👍